### ISSUE TITLE
Add. 14,668, fol. 46 | https://bl.vuexistapps.us/manuscript/495 (Part12)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1122]
### Problems with Manuscript Overview

- A sentence in the printed catalog, missing from the online version, informs that there is the shape of a cross at the end of the leave. So, maybe the Decoration section should also be added. 

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

### Problems with Physical Description

- "The writing is small and regular." repeats both in the main paragraph of Phys. Des. and in Hand 1. 
- "It is dated A. Gr. 1199, A.D. 888" appears in the main paragraph of Phys. Des. but perhaps it should be placed in Hand 1, since that is where the date is usually mentioned. 
- Hand 2 refers to Addition 2. However, it seems that there is no clear evidence that the note was written by a different hand. 
- The following information is missing: "Beneath, there is drawn a cross." 

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [X] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
Resolved 
- Repeated hand info deleted
- Date added to hand 1
- Decoration added

Question
Should we keep hand 2? @wlpotter

**Comment by wlpotter** [comment]
Yes, let's keep it (Wright calls it "a later note", suggesting it's a separate hand from the rest of the leaf)
